src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -494,7 +491,7 @@ public void testServiceCheckCounter() throws Exception {
         // Let's put a service check in the pipeline (we cannot call doIteration()
         // here unfortunately because it would call reportStatus which will flush
         // the count to the jmx_status.yaml file and reset the counter.
-        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", "jmx_test_instance", null);
+        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", null);
 
         // Let's check that the counter has been updated
         assertEquals(1, repo.getServiceCheckCount("jmx"));
```

Comment:
Can we test the "jmx_server: hostname" tag ? In the meantime, maybe we can start moving those service checks tests in a different file to improve the readibility.